python/ray/tune/schedulers/pbt.py

```diff
@@ -49,8 +49,9 @@ def _explore(
     config: Dict,
     mutations: Dict,
     resample_probability: float,
+    perturbation_factors: Tuple[float],
     custom_explore_fn: Optional[Callable],
-) -> Dict:
+) -> Tuple[Dict, Dict]:
     """Return a config perturbed as specified.
```

Comment:
update?

Reply:
What should be updated? The return type should be updated already.